assets/search/header.ejs

```diff
@@ -5,13 +5,15 @@
       ng-submit="search.submit()">
   <input type="submit" style="position: absolute; left: -9999px"/>
 
-  <div class="form-group" ng-if="!onSearchPage && user">
+  <div class="form-group" ng-if="user">
     <div class="form-group has-feedback">
       <div class="input-group">
         <select class="form-control input-group-addon"
                 ng-model="search.input.search"
+                ng-change="search.toggleCategory()"
```

Comment:
Why did you put this here? This is for the ref search, for the categories that we have.